includes/class-wcs-exporter-cron.php

```diff
@@ -130,14 +130,13 @@ public static function filter_payment_method( $query_args, $args ) {
 	}
 
     /**
-	 * Delete subscription export file.
+	 * Delete subscription export file, including the tmp file if it exists.
 	 *
 	 * @since 2.0-beta
 	 * @param string $file_name
 	 * @return array
```

Comment:
Function is not returning anything.

Reply:
Thanks @menakas :) good eye!

That `@return array` was added by an earlier contributor and not introduced by this PR so I think we should just leave it for now and fix it up another time depending on whether we want to address all other phpcs issues.

FWIW, this file has lots of issues like this one, including whitespace issues (i.e. the whole file uses a combination of spaces and tabs) and incorrect docblocks.